packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFGeneratedWebKitApis.h

```diff
@@ -0,0 +1,426 @@
+
+@protocol FWFUIViewHostApi
+/// @return `nil` only when `error != nil`.
+- (nullable NSArray<NSNumber *> *)getContentOffsetInstanceId:(NSNumber *)instanceId
+                                                       error:
+                                                           (FlutterError *_Nullable *_Nonnull)error;
+- (void)setBackgroundColorInstanceId:(NSNumber *)instanceId
+                               value:(nullable NSNumber *)value
+                               error:(FlutterError *_Nullable *_Nonnull)error;
+- (void)setOpaqueInstanceId:(NSNumber *)instanceId
+                     opaque:(NSNumber *)opaque
+                      error:(FlutterError *_Nullable *_Nonnull)error;
+@end
+
+extern void FWFUIViewHostApiSetup(id<FlutterBinaryMessenger> binaryMessenger,
+                                  NSObject<FWFUIViewHostApi> *_Nullable api);
+
+/// The codec used by FWFUIScrollViewHostApi.
+NSObject<FlutterMessageCodec> *FWFUIScrollViewHostApiGetCodec(void);
+
+@protocol FWFUIScrollViewHostApi
+- (void)createFromWebViewInstanceId:(NSNumber *)instanceId
+                  webViewInstanceId:(NSNumber *)webViewInstanceId
+                              error:(FlutterError *_Nullable *_Nonnull)error;
+/// @return `nil` only when `error != nil`.
+- (nullable NSArray<NSNumber *> *)getContentOffsetInstanceId:(NSNumber *)instanceId
+                                                       error:
+                                                           (FlutterError *_Nullable *_Nonnull)error;
+- (void)scrollByInstanceId:(NSNumber *)instanceId
+                         x:(NSNumber *)x
+                         y:(NSNumber *)y
+                     error:(FlutterError *_Nullable *_Nonnull)error;
+- (void)setContentOffsetInstanceId:(NSNumber *)instanceId
+                                 x:(NSNumber *)x
+                                 y:(NSNumber *)y
+                             error:(FlutterError *_Nullable *_Nonnull)error;
+@end
+
+extern void FWFUIScrollViewHostApiSetup(id<FlutterBinaryMessenger> binaryMessenger,
+                                        NSObject<FWFUIScrollViewHostApi> *_Nullable api);
+
+/// The codec used by FWFWKWebViewConfigurationHostApi.
+NSObject<FlutterMessageCodec> *FWFWKWebViewConfigurationHostApiGetCodec(void);
+
+@protocol FWFWKWebViewConfigurationHostApi
+- (void)createInstanceId:(NSNumber *)instanceId error:(FlutterError *_Nullable *_Nonnull)error;
+- (void)createFromWebViewInstanceId:(NSNumber *)instanceId
+                  webViewInstanceId:(NSNumber *)webViewInstanceId
+                              error:(FlutterError *_Nullable *_Nonnull)error;
+- (void)setAllowsInlineMediaPlaybackInstanceId:(NSNumber *)instanceId
+                                         allow:(NSNumber *)allow
+                                         error:(FlutterError *_Nullable *_Nonnull)error;
+- (void)
+    setMediaTypesRequiringUserActionForPlaybackInstanceId:(NSNumber *)instanceId
```

Comment:
setMediaRequiresUserActionForConfigurationWithIdentifier:forTypes:error:`

(The double `for` is a bit weird there, but it's better than omiting it IMO given the ordering requirement.)